tests/test_views.py

```diff
@@ -44,4 +42,3 @@ def test_post_detail_response_with_correct_template(self):
 
         self.assertEqual(response.status_code, 200)
         self.assertTemplateUsed(response, "blog/post_detail.html")
```

Comment:
Ensure the template name in the `assertTemplateUsed` method is corrected to 'blog/post-detail.html' to match the previous feedback.